src/main/scala/com/dwolla/cloudflare/DnsRecordClient.scala

```diff
@@ -13,10 +13,10 @@ import scala.language.implicitConversions
 
 class DnsRecordClient(executor: CloudflareApiExecutor)(implicit val ec: ExecutionContext) {
 
-  import Implicits._
+  import com.dwolla.cloudflare.domain.model.Implicits._
 
-  protected lazy val logger: Logger = LoggerFactory.getLogger("LambdaLogger")
-  protected implicit val formats: Formats = DefaultFormats ++ org.json4s.ext.JodaTimeSerializers.all
+  protected lazy val logger: Logger = LoggerFactory.getLogger(classOf[DnsRecordClient])
```

Comment:
If this is going to be used in AWS Lambdas, it should use the `"LambdaLogger"` logger, so that CloudWatch logging doesn't split the log stream into multiple log events using newlines.